### PR TITLE
Fixes #18

### LINKED ISSUE #18: Create a sign-up form for subscribers
**Introduction**
We require a simple input field and button to allow users to subscribe to our multiple subscription lists. Our API, located in https://mailchimp.mybit.io, receives a POST call to the endpoint "api/member" receiving an email and a list to subscribe a user to. You can see the code of our API in the following link - https://github.com/MyBitFoundation/mailchimp-proxy/blob/master/index.js.

Using our UI Kit (https://ui.mybit.io), create a React component that handles all the possible uses cases for signing up a user (some validation errors are returned by the server, as can be seen in the code) to a given list using our API. The idea is to be able to define the list name in the front-end and reuse the component across our multiple websites and projects.

You can test the API using Postman and generating a POST call with your data.

https://github.com/MyBitFoundation/mailchimp-proxy/blob/master/index.js

const express = require('express');
const Mailchimp = require('mailchimp-api-v3');
const validator = require("email-validator");

require('dotenv').config();

Resources:
https://files.mybit.io/files/

**To submit please create a pull request.**

Disclaimer
All work completed via the MyBit Decentralised Development Fund is property of the MyBit Foundation (CHE-177.186.963) and may be used, re-used, and/or distributed, re-distributed by the MyBit Foundation for financial gain. The contributor holds no warrant nor claims for any future payments resulting from monetisation schemes originating from the MyBit Foundation or any of its affiliates. Where law permits outside of open source regulations, the contributor is not permitted to distribute any work completed via the MyBit Decentralised Development Fund (DDF) for personal gain.

### PR DESCRIPTION
**Pull request by uivlis**
Created subscription form

### CODE REVIEW

File: src/showcase/SubscribeForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  handleButtonClick(e){
+    console.log(this.state.value);
+    if (!this.validateEmail(this.state.value)){
+      this.setState({
+        value: '',
+        placeholder: 'Invalid email. Try again'
+      });
+    } else {
+        var xhr = new XMLHttpRequest();
+        xhr.open("POST", 'https://cors-anywhere.herokuapp.com/https://mailchimp.mybit.io/api/member/', true);
```
Comment (csmartinsfct) [inline src/showcase/SubscribeForm/index.jsx]: Haven't tested the code yet but I'm guessing this URL is wrong :)

Comment (uivlis) [reply src/showcase/SubscribeForm/index.jsx]: It isn't :). For me, it works.
Apparently, I needed to disable CORS, and that is impossible via { mode: 'no-cors' } in the browser, so I used a proxy-url.

Comment (csmartinsfct) [reply src/showcase/SubscribeForm/index.jsx]: Oh I see, but we can't merge this as is.

Comment (uivlis) [reply src/showcase/SubscribeForm/index.jsx]: I don't see another way. CORS cannot be set from the browser, so another alternative would be to modify your [server](https://github.com/MyBitFoundation/mailchimp-proxy/blob/master/index.js).

Comment (jjperezaguinaga) [reply src/showcase/SubscribeForm/index.jsx]: Thanks for the update! No need to disable CORS, we can probably have a staging URL that doesn’t has it, but the component should only be used in *.mybit.io domains. My suggestion is to have the URL passed down as a parameter or set out in a dynamic way depending in a env variable like `NODE_ENV`

Comment (jjperezaguinaga) [reply src/showcase/SubscribeForm/index.jsx]: Thanks! @csmartinsfct can you take a look at this and see if we can test it locally?

Comment (uivlis) [reply src/showcase/SubscribeForm/index.jsx]: Hey @csmartinsfct did you manage to test this? Also, @jjperezaguinaga, unfortunately, I didn't receive the bounty for https://github.com/MyBitFoundation/MyBit-Network.tech/issues/151 (the pull request had "Fixes #151" in the description). Is there any way to get the lost MYB's?

Comment (jjperezaguinaga) [reply src/showcase/SubscribeForm/index.jsx]: 👋 @uivlis Apologies for that! Just commented on that issue, pretty much open a PR with an empty commit, so you get access to the bounty ☺️

**Comment by jjperezaguinaga** [comment]
@uivlis We are still evaluating our CI/CD to preview this inputs within our infrastructure. Please hang on while we clean it up!

**Comment by jjperezaguinaga** [comment]
Looking good! https://pull-42-ui-mybit-io-dev.now.sh/subscribe-form We'll take it from here ☺️

**Pull request merged**